internal/config/config.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/google/osv-scanner/v2/internal/imodels"
 )
 
-const osvScannerConfigName = "osv-scanner.toml"
+var OsvScannerConfigName = "osv-scanner.toml"
```

Comment:
`OSVScannerConfigName`

Reply:
isn't it go convention to not do this? technically we can do it because the linter only enforces a built-in list, but it'll be inconsistent with other stuff like Id, Api, Json, etc

Reply:
oh yeah whoops its the otherway around